CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKRenderable.java

```diff
@@ -146,25 +147,16 @@ public int SetNewSwapChain(CVKSwapChain newSwapChain) {
     
     public abstract void IncrementDescriptorTypeRequirements(CVKDescriptorPoolRequirements reqs, CVKDescriptorPoolRequirements perImageReqs);
     public abstract int RecordCommandBuffer(VkCommandBufferInheritanceInfo inheritanceInfo, int index);
```

Comment:
This should be renamed to indicate what it's recording to.

Reply:
I think it is fine for now, can revisit later if need

Reply:
Please change in a future changelist.  The vulkan display module is huge and probably confusing to anyone who hasn't spent ages looking at it.  I think super clear function names are a necessity.  RecordDisplayCommandBuffer and RecordHitTestCommandBuffer are nice clear names to me.

Reply:
Ok I've renamed them